mint/fakeAPIData.ts

```diff
@@ -203,17 +205,46 @@ export const wideData = [
     }
 ];
 
-export const columns =
-    {
-        "intervention": "Intervention",
-        "net_use": "Net use",
-        "prev_year_1": "Prevalence Under 5 yrs: Yr 1 post intervention",
-        "prev_year_2": "Prevalence Under 5 yrs: Yr 2 post intervention",
-        "prev_year_3": "Prevalence Under 5 yrs: Yr 3 post intervention",
-        "cases_averted": "Cases averted across 3 yrs since intervention",
-        "cases_averted_high": "Upper bound for cases averted",
-        "cases_averted_low": "Lower bound for cases averted"
-    };
+export const tableConfig: ColumnDefinition[] =
```

Comment:
This is the proposed config format - each column has a `valueCol` and `displayName`, with further optional properties  `valueTransform` for transforming the value in the specified col which I figured could be used both for specifying intervention display names (since in the real data they're things like "llin-pbo" etc), and for the actual mathematical calculations, and `format`, which would just take a `numeraljs` format like we've done in HINT.